src/main/java/com/faforever/client/chat/ChatUserControl.java

```diff
@@ -48,8 +82,13 @@
   @FXML
   Label clanLabel;
 
+  @FXML
+  ImageView statusImageView;
+
   private PlayerInfoBean playerInfoBean;
 
+  private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
static fields always go to the top (enabling "rearrange code" takes care of this)